src/classes/Stringy.ts

```diff
@@ -20,4 +20,22 @@ export default class Stringy {
 
         return refinedPrefix.length === matches.length;
     }
+
+    isSuffix(suffix: string, isCaseSensitive: boolean = false) {
```

Comment:
Codacy has a fix for the issue: [Delete `··`](https://app.codacy.com/gh/VictorMaria/Strings-and-Arrays/pullRequest?prid=9807711)
```suggestion
  isSuffix(suffix: string, isCaseSensitive: boolean = false) {
```